packages/lab/src/dropdown-next/DropdownNext.tsx

```diff
@@ -0,0 +1,256 @@
+import { clsx } from "clsx";
+import { ListNext, ListNextProps } from "../list-next";
+import {
+  makePrefixer,
+  useId,
+  useForkRef,
+  UseFloatingUIProps,
+  SaltProvider,
+} from "@salt-ds/core";
+import { ChevronDownIcon, ChevronUpIcon } from "@salt-ds/icons";
+import {
+  useRef,
+  forwardRef,
+  HTMLAttributes,
+  FocusEvent,
+  KeyboardEvent,
+  MouseEvent,
+  MutableRefObject,
+} from "react";
+import { useWindow } from "@salt-ds/window";
+import dropdownNextCss from "./DropdownNext.css";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { FloatingPortal, Placement } from "@floating-ui/react";
+import { useDropdownNext } from "./useDropdownNext";
+
+const withBaseName = makePrefixer("saltDropdownNext");
+
+export interface DropdownNextProps
+  extends Pick<UseFloatingUIProps, "open" | "onOpenChange" | "placement">,
+    HTMLAttributes<HTMLElement> {
+  /**
+   * If `true`, dropdown will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Initially selected value for the dropdown.
+   */
+  defaultSelected?: string;
+  /**
+   * List of options when using a dropdown.
+   */
+  source: string[];
+  /**
+   * If `true`, dropdown is read only.
+   */
+  readOnly?: boolean;
+  /**
+   * Background styling variant. Defaults to `primary` .
+   */
+  variant?: "primary" | "secondary";
+  /**
+   * Placement of dropdown list. Defaults to `bottom` .
+   */
+  placement?: Placement;
+  /**
+   * Additional props for dropdown list.
+   */
+  ListProps?: ListNextProps;
+
+  /* Status open or close for use in controlled component.  */
+  open?: boolean;
+  /* Selected item prop for use in controlled component. */
+  selectedItem?: string;
+  /* Highlighted item prop for use in controlled component. */
+  highlightedItem?: string;
+  /* Callback for blur event. */
+  onBlur?: (event: FocusEvent) => void;
+  /* Callback for focus event. */
+  onFocus?: (event: FocusEvent) => void;
+  /* Callback for mouse over event. */
+  onMouseOver?: (event: MouseEvent) => void;
+  /* Callback for mouse down event. */
+  onMouseDown?: (event: MouseEvent) => void;
+  /* Callback for key down event. */
+  onKeyDown?: (event: KeyboardEvent) => void;
+}
+
+export const DropdownNext = forwardRef<HTMLDivElement, DropdownNextProps>(
+  function DropdownNext(props, ref) {
+    const {
+      className,
+      disabled,
+      variant = "primary",
+      id: dropdownIdProp,
+      defaultSelected,
+      readOnly,
+      source,
+      placement = "bottom",
+      open: openControlProp,
```

Comment:
should the open prop be set to false as default, or is this managed in the useControlled hook ?

Reply:
it's managed in the useControlled hook :)